### PR TITLE
Added  a table the details /Exersize

### PR DESCRIPTION
**Pull request by Aaliyah379**
Please review 
Thank you

### CODE REVIEW

File: MyWorkout.Web/Controllers/ExercisesController.cs [hunk 1]
```diff
@@ -22,6 +22,23 @@ public ExercisesController(IUnitOfWork unitOfWork)
         }
 
         // GET: Exercises
+        //Exersices/CreateFromDayOfWeek/1
+
+        public IActionResult CreateFromDayOfWeek(int? id)
+        {
+            if (id == default(int))
+            {
+                throw new ArgumentNullException(" Day of week id is not set");
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/ExercisesController.cs]: Remove please white space before " Day of week"

File: MyWorkout.Web/Views/Plans/Details.cshtml [hunk 2]
```diff
@@ -24,6 +24,7 @@
     <table class="table">
         <thead>
             <tr>
+                <a href="C:\Users\robby\source\repos\YuriyMorozyuk95\MyWorkout\MyWorkout.Web\Views\Plans\Details.cshtml">C:\Users\robby\source\repos\YuriyMorozyuk95\MyWorkout\MyWorkout.Web\Views\Plans\Details.cshtml</a>
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Views/Plans/Details.cshtml]: please remove this =)

Comment (Aaliyah379) [reply MyWorkout.Web/Views/Plans/Details.cshtml]: fixed accordingly

File: MyWorkout.Web/Controllers/ExercisesController.cs (hunk 1 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/ExercisesController.cs]: space has been removed

**Pull request merged**